### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by ehanna4**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,9 +52,12 @@ e.g. `sbt "resourceValidator/run --dryRun --all"`
...
 
-Once a PR is merged, there will be a PR created in [terra-helm](https://github.com/broadinstitute/terra-helm). 
-Get this PR merged, and another automatic commit will bump leonardo's chart version. This will trigger another automatic commit 
-in [terra-helmfile](https://github.com/broadinstitute/terra-helmfile), note this commit will only auto bump `dev` and `perf`, and will be auto-merged. Once the terra-helmfile PR is auto-merged, go to [argo](https://ap-argocd.dsp-devops.broadinstitute.org/applications) (you need to be on VPN to access argo), and click `SYNC APPS` button on the left upper corner for dev and perf leonardo deploys (select `PRUNE` option as well).
+Once a PR is merged, a PR will be automatically created in [terra-helm](https://github.com/broadinstitute/terra-helm). 
+
+Get this PR merged (you can merge it yourself) and another automatic commit will bump leonardo's chart version. This will trigger another automatic commit 
+in [terra-helmfile](https://github.com/broadinstitute/terra-helmfile), note this commit will only auto bump `dev` and `perf`, and will be auto-merged.
```
Comment (kyuksel) [inline README.md:58]: Nit:
```suggestion
in [terra-helmfile](https://github.com/broadinstitute/terra-helmfile). Note that this commit will only auto-bump `dev` and `perf`, and will be auto-merged.
```

**Review by kyuksel — APPROVED** [review]
Thanks for taking the time to update the README!  👍🏼  with a nit inline.

File: README.md (hunk 1 above)
Comment (ehanna4) [reply README.md:58]: oh my you were fast to review! haha I am hoping to make a few more edits, will update you once i commit those!

Comment (kyuksel) [reply README.md:58]: my bad that I jumped on it despite the fact that you had marked it as `Draft` 🤦🏼

Comment (ehanna4) [reply README.md]: no problem! appreciate the feedback!

**Review by kyuksel — APPROVED** [review]
Looks great! 👍🏼

**Pull request merged**